ogre/src/OgreLidarVisual.cc

```diff
@@ -171,13 +192,38 @@ void OgreLidarVisual::Update()
   // Process each point from received data
   // Every line segment, and every triangle is saved separately,
   // as a pointer to a DynamicLine
+  // This initializes and updates only the selected DynamicLine variables
   for (unsigned int j = 0; j < this->verticalCount; ++j)
   {
     horizontalAngle = this->minHorizontalAngle;
 
-    if (j+1 > this->dataPtr->rayLines.size())
+    if (this->dataPtr->lidarVisType == LidarVisualType::LVT_RAY_LINES)
     {
-      // Ray Strips fill in between the line areas that intersect an object
+      if (j+1 > this->dataPtr->rayLines.size())
+      {
+        std::shared_ptr<OgreDynamicLines> line =
+                    std::shared_ptr<OgreDynamicLines>(
+                          new OgreDynamicLines(MT_LINE_LIST));
+
+        #if (OGRE_VERSION <= ((1 << 16) | (10 << 8) | 7))
+              line->setMaterial("Lidar/BlueRay");
+        #else
+              line->setMaterial(
+                  this->Scene()->Material("Lidar/BlueRay")->Clone());
```

Comment:
You'll need to use `Ogre::MaterialManager` like [here](https://github.com/ignitionrobotics/ign-rendering/pull/103/files#diff-774fb867ada2642609f3037231117bb2R188)